kickstarts/base.ks.erb

```diff
@@ -99,6 +99,9 @@ ramfsfile="/boot/initramfs-$kversion.img"
 # make sure there is a new line at the end of sshd_config
 echo "" >> /etc/ssh/sshd_config
 
+# Clean the logs
+rm -rf "$vmdb_root/log/*"
```

Comment:
I just noticed this, and I think we should use the trick that Geert used where we mount tmpfs, and then just toss the tmpfs disk at the end.  This way we keep the image size down since deletes don't really delete the on-disk bits. cc @simaishi